packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/vsphere-users.constant.js

```diff
@@ -2,6 +2,13 @@ export const DEFAULT_FILTER_COLUMN = 'name';
 
 export const ENUM_USERS_STATUS = ['creating', 'deleting', 'delivered', 'error'];
 
+export const IAM_STATUS_ENABLED = 'enabled';
+
+export const USER_IDENTITY_PROVIDER_IAM = 'iam';
+
+export const USER_TYPE_USER = 'user';
```

Comment:
**nitpick:** Variable name is confusing `USER_TYPE_USER` Is it possible to change it?

Reply:
Any suggestion ? 
user type can be one of the following : user, role or group